infrastructure/main.tf

```diff
@@ -27,9 +28,9 @@ module "bulk-scan" {
   capacity     = "${var.capacity}"
 
   app_settings = {
-    STORAGE_ACCOUNT_NAME = "${azurerm_storage_account.provider.name}"
-    STORAGE_KEY          = "${azurerm_storage_account.provider.primary_access_key}"
-
+    STORAGE_ACCOUNT_NAME    = "${azurerm_storage_account.provider.name}"
+    STORAGE_KEY             = "${azurerm_storage_account.provider.primary_access_key}"
+    SSCS_SAS_TOKEN_VALIDITY = "${local.tokenValidity}"
```

Comment:
The name suggests that the variable is related to SSCS only. Is that the case?

Reply:
yes will update variable name.